pytorch_lightning/loggers/tensorboard.py

```diff
@@ -195,8 +197,8 @@ def log_hyperparams(
             self.hparams.update(params)
 
         # format params into the suitable for tensorboard
-        params = self._flatten_dict(params)
-        params = self._sanitize_params(params)
```

Comment:
Why not using self here anymore?

Reply:
self is correct here, it has been fixed now.